### ISSUE TITLE
Enhance visibility of the active tab in the default color scheme

### ISSUE THREAD

**Issue by zuglufttier** [qutebrowser/qutebrowser#3072]
Regardless of the version, the default colors have, in my eyes, one big problem: The currently active tab is highlighted by a black color.

The human eye is very good at differentiating shades of grey (more than 50), so the current default color scheme makes it pretty easy to distinguish the different tabs without relying on something like outlining. 

This is the good part, the bad part is the following: The eye is always drawn to the light parts of a picture first, so in the default color scheme you always recognize the inactive tabs first and only then realize that the active tab is the black one. So, I suggest that the default scheme uses a very light grey for the active tab and two rather dark greys for the inactive tabs.

Yes, I could make my own config and be happy but a sane default configuration is always a good thing.
This would be my suggestion for the default settings:

```
:set colors.tabs.selected.odd.fg black
:set colors.tabs.selected.even.fg black
:set colors.tabs.selected.odd.bg #eeeeee
:set colors.tabs.selected.even.bg #eeeeee
```

**Comment by The-Compiler** [comment]
I'm going to say "no" to this one, sorry - I agree good defaults are important, but I'd hate to significantly change the way qutebrowser looks by default, and at least I've had a much harder time intuitively/quickly finding the active tab with the change you propose.

**Comment by Ape** [comment]
I had this idea of tab color reversal earlier myself. Finally I decided to just use the black background for the selected tab (same as the default).

My reasoning was that it simply looks better if you have a black statusbar at the bottom of the window. I looks more consistent, especially if there is only one tab.

**Comment by Hi-Angel** [comment]
@Ape don't you confuse the focused tab because of unfocused ones alternating two shades of grey? *(again, by default)*

**Comment by Ape** [comment]
I remember it was a problem in the past. I tried to fix it with this: #2864. However, I am not using that currently anymore. Instead, I adjusted the colors a bit. I don't have any issues recognizing the focused tab. This is what I currently use (config.py format):

```python
# Background color of unselected tabs.
c.colors.tabs.even.bg = "silver"
c.colors.tabs.odd.bg = "gainsboro"

# Foreground color of unselected tabs.
c.colors.tabs.even.fg = "#666666"
c.colors.tabs.odd.fg = c.colors.tabs.even.fg
```

**Comment by Hi-Angel** [comment]
@Ape so the default tab colors have caused a problem until you changed it. Better default experience is exactly what we're talking about. 

@The-Compiler ↑
